test/e2e/alertmanager_e2e_test.go

```diff
@@ -230,3 +230,81 @@ func TestMeshInitialization(t *testing.T) {
 		}
 	}
 }
+
+func TestAlertmanagerReloadConfig(t *testing.T) {
+	alertmanager := framework.MakeBasicAlertmanager("reload-config", 1)
+	alertmanagerService := framework.MakeAlertmanagerService(alertmanager.Name, "alertmanager-service", v1.ServiceTypeClusterIP)
```

Comment:
This `Service` is unused as we request the pod directly through the proxy, so we can remove it. :tada: